index.js

```diff
@@ -342,38 +337,50 @@ module.exports = {
             // prettier-ignore
             if (err) return cb(clarify(err, 'Failed to get root metafeed when listing invites'))
 
-            ssb.box2.listGroupIds((err, groupIds) => {
-              // prettier-ignore
-              if (err) return cb(clarify(err, 'Failed to list group IDs when listing invites'))
-
-              const source = pull(
-                ssb.db.query(
-                  where(and(isDecrypted('box2'), type('group/add-member'))),
-                  toPullStream()
-                ),
-                pull.filter((msg) =>
-                  // it's an addition of us
-                  lodashGet(msg, 'value.content.recps', []).includes(myRoot.id)
-                ),
-                pull.filter(
-                  (msg) =>
-                    // we haven't already accepted the addition
-                    !groupIds.includes(lodashGet(msg, 'value.content.recps[0]'))
-                ),
-                pull.map((msg) => {
-                  return {
-                    id: lodashGet(msg, 'value.content.recps[0]'),
-                    secret: Buffer.from(
-                      lodashGet(msg, 'value.content.secret'),
-                      'base64'
-                    ),
-                    root: lodashGet(msg, 'value.content.root'),
-                  }
-                })
-              )
-
-              return cb(null, source)
-            })
+            pull(
+              ssb.box2.listGroupIds(),
+              pull.collect((err, groupIds) => {
+                if (err)
```

Comment:
Could you bring back the `prettier-ignore` for this line? It's a pattern we use in ssb-db2 too, to consistently have the error line not take up that much space.

Reply:
i think this got lost in a merge. i'll fix